viewer/src/locales/Italian.js

```diff
@@ -0,0 +1,189 @@
+/**
+ * Some phrases contain insertions, e.g. icons and buttons, which are inserted in the code.
+ * Here instead of visual components we use placeholders, e.g. #helpButton, which start with #.
+ * Your translated phrase MUST also contain the same placeholder, but you can change its position.
+ *
+ * Some phrases are tooltips, that is, they are visible only when you hover the cursor over controls.
+ *
+ * And try to preserve the order of phrases (just for convenience of further additions and corrections).
+ */
+ 
+export default {
+    // language info
+    englishName: "Italian",
+    nativeName: "Italiano",
+
+    "Language": "Lingua", // not used now, but will be used in options afterwards
+
+    // Translation: tooltips and notification
+    // (to see the notification window, remove several phrases from any dictionary, except for the English one)
+    "Add more":
+        "Aggiungi traduzione",
+    "The translation isn't complete.":
+        "La traduzione non è completa",
+    "The following phrases are not translated:":
+        "Le seguenti frasi non sono tradotte:",
+    "You can improve the translation here":
+        "Migliora la traduzione qui",
+
+    // Initial screen
+    "#helpButton - learn more about the app":
+        "#helpButton - Istruzioni all'uso dell'app",
+    "#optionsButton - see the available options":
+        "#optionsButton - Opzioni disponibili",
+    "powered with":
+        "Powered with",
```

Comment:
Is it intentional? Or you just forgot about this line?

Reply:
Intentional, is cooler said in English :-)